### PR TITLE
Target netstandard2.0 as well as netcoreapp2.2 

### PR DESCRIPTION
**Pull request by RozzaysRed**
Addresses #209

### CODE REVIEW

**Comment by longfin** [comment]
I think it needs to mention this change in the changelog.

File: CHANGES.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ To be released.
 
  -  Improved performance of `StoreExtension.LookupStateReference<T>()` method.
     [[#447], [#545]]
+ -  Added .Net Core 2.2 as a targeted framework. [[#561]]
```
Comment (moreal) [inline CHANGES.md]:
```suggestion
 -  Added .NET Core 2.2 as a targeted framework.  [[#209], [#561]]
```

File: CHANGES.md [hunk 2]
```diff
@@ -44,6 +45,7 @@ To be released.
 [#560]: https://github.com/planetarium/libplanet/pull/560
 [#562]: https://github.com/planetarium/libplanet/pull/562
 [#563]: https://github.com/planetarium/libplanet/pull/563
+[#561]: https://github.com/planetarium/libplanet/pull/561
```
Comment (moreal) [inline CHANGES.md]:
```suggestion
[#209]: https://github.com/planetarium/libplanet/issues/209
[#561]: https://github.com/planetarium/libplanet/pull/561
```

It was missed.

**Comment by longfin** [comment]
@RozzaysRed Thanks for your contribution!

**Pull request merged**